src/components/viewers/MultiFlatmap.vue

```diff
@@ -24,6 +24,35 @@ import EventBus from "../EventBus";
 import store from "../../store";
 import markerZoomLevels from '../markerZoomLevels';
 
+/*
+ * Function to check markers visibility at zoom level.
+ * I have modified it to make sure the marker is displayed
+ * if the uberon is not present in the hardcoded zoom-level list.
+ */
+
+const checkMarkersAtZoomLevel = (flatmapImp, markers, zoomLevel) => {
+  if (markers) {
+    markers.forEach(id=>{
+      let foundInArray = false;
+      //First check if uberon is in the list, check for zoom level 
+      //if true. Note: markerZoomLevels is imported.
+      for (let i = 0; i < markerZoomLevels.length; i++) {
```

Comment:
I personally prefer `++i` to `i++` since it usually results in slightly (ok, negligibly) faster code.